### PR TITLE
44702 Thousand separator in form inputs

### PR DESCRIPTION
**Pull request by dai-eastgate**
related to https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/721
changed log:
Add thousand separator in form inputs

### CODE REVIEW

File: js/onoffice-apply-thousand-separator.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+onoffice_setting = onoffice_setting || [];
+jQuery(document).ready(($) => {
+    const thousandSeparatorFormat = onoffice_setting.thousand_separator_format || '';
+
+    const getSeparators = (separatorFormat) => ({
+        thousandSeparator: separatorFormat === 'comma-separator' ? ',' : '.',
+        decimalSeparator: separatorFormat === 'comma-separator' ? '.' : ','
```
Comment (andernath) [inline js/onoffice-apply-thousand-separator.js:7]: In https://github.com/onOffice-Web-Org/oo-wp-plugin/issues/721#issuecomment-1966093746 you mention that you did not implement decimal separator. 
Does onoffice-apply-thousand-separator.js need some refactoring ?

**Review by andernath — COMMENTED** [review]
@dai-eastgate looks good. I just have one question, could you take a look?

**Comment by dai-eastgate** [comment]
@fredericalpers  @andernath I removed the decimal separator implementation
=> Currently the thousands separator feature will be automatic, users cannot enter "," or "." 
Ex:
With dot: 1000 -> 1.000; 123456 -> 123.456
With comma: 1000 -> 1,000; 123456 -> 123,456

Note: I have added a "Please choose" option for old users who are not using this feature

You can watch the demo video below:

https://github.com/user-attachments/assets/39a7ce94-eca8-4289-8d1a-cef8d1c157d4

**Review by miauzjpg — APPROVED** [review]
great work!

**Pull request merged**